test/e2e/exists_test.go

```diff
@@ -82,4 +82,36 @@ var _ = Describe("Podman image|container exists", func() {
 		Expect(session.ExitCode()).To(Equal(1))
 	})
 
+	It("podman pod exists in local storage by name", func() {
+		setup, rc, _ := podmanTest.CreatePod("foobar")
+		setup.WaitWithDefaultTimeout()
+		Expect(rc).To(Equal(0))
+
+		session := podmanTest.Podman([]string{"pod", "exists", "foobar"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+	})
+	It("podman pod exists in local storage by container ID", func() {
+		setup, rc, podID := podmanTest.CreatePod("")
+		setup.WaitWithDefaultTimeout()
+		Expect(rc).To(Equal(0))
+
+		session := podmanTest.Podman([]string{"pod", "exists", podID})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+	})
+	It("podman container exists in local storage by short container ID", func() {
```

Comment:
I think the test names are a cut/paste error here and for the next test.

Reply:
@baude I don't think you actually fixed this - did you push?